accounts-cluster-bench/src/main.rs

```diff
@@ -130,6 +134,30 @@ struct SeedTracker {
     max_closed: Arc<AtomicU64>,
 }
 
+type TransactionSignatureTracker = Arc<RwLock<VecDeque<Signature>>>;
```

Comment:
nit: I'd `struct TransactionSignatureTracker(Arc<RwLock<...>)`

it's not very idiomatic to use a trait for this. Also always add an empty
newline between item definitions